README.md

```diff
@@ -2,24 +2,42 @@
 
-# Get it from Google Play
-
-[![Rocket.Chat on Google Play](https://user-images.githubusercontent.com/551004/29770692-a20975c6-8bc6-11e7-8ab0-1cde275496e0.png)](https://play.google.com/store/apps/details?id=chat.rocket.android)
+## Description
+Currently, the app is maintained in two branches, namely `v1+` and `v2+`. The `v1+` is maintained in the `develop` branch and the `v2+` is maintained in the `develop-2.x` branch. The older version is written partially in `java` and `kotlin`, but we intend to write the latest version completely in `kotlin`.
 
 ## How to build
-
-Retrolambda needs java8 to be installed on your system
+Clone the repository by running `git clone https://github.com/RocketChat/Rocket.Chat.Android.git` in your terminal. To build the v1.0+ of the app, run `git checkout develop` and to build the v2.0+, run `git checkout develop-2.x`. 
+Since both the versions use `kotlin` for some or all of their classes, following are the common prerequisites for both versions:
+- Android Studio 3.0+ comes with built in kotlin support, so install the latest version (3.0+) of Android Studio (Recommended). For older versions, you need to manually install kotlin plugin. Go to `File>Settings>Plugins` and search for `kotlin` and install it. You'll need to restart the IDE in order to see changes.
+- Make sure that you have the latest **gradle** and the **android plugin** versions installed. Go to `File>Project Structure>Project` and make sure that you have the latest versions installed. Refer [this](https://developer.android.com/studio/releases/gradle-plugin.html#updating-gradle) to see the compatible versions.
+- Kotlin is already configured in the project. To check, go to `Tools>Kotlin>Configure Kotlin in project`. A message saying kotlin is already configured in the project pops up. You can update kotlin to the latest version by going to `Tools>Kotlin>Configure Kotlin updates` and download the latest version of kotlin.
+
+### Instructions specific to version
+#### v1.0+ 
+- After checking out to `develop` branch as mentioned above, simply import the project in Android Studio.
+
+#### v2+
+- This version requires the [Kotlin SDK](https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK) for Rocket.Chat. Clone the Kotlin SDK in the **same directory** as the android repository by running `git clone https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK.git`. Make sure that the android repository and the kotlin sdk have the same immediate parent directory.
+- First, a build is required for the SDK. Change your directory to the SDK directory by running `cd Rocket.Chat.Kotlin.SDK/` in your terminal. Any of the following approaches can be followed to successfully build the SDK.
+    - **Command Line** - (Within the kotlin SDK directory) Run `./gradlew clean&&./gradlew assemble` to successfully build the project.
```

Comment:
Please change `./gradlew clean&&./gradlew assemble` to `./gradlew clean && ./gradlew assemble`